File: driver-api/src/main/kotlin/org/tonycox/garage/driver/api/model/TripInfo.kt
package org.tonycox.garage.driver.api.model

import java.time.LocalTime

/**
 * @author Anton Solovev
 * @since 3/15/2018.
 */
data class TripInfo(val driverId: Long,
                    val path: List<String>,
                    val sitCount: Long,
                    val departureTime: LocalTime,
                    val arriveTime: LocalTime)

File: driver-data-ingester/src/main/scala/org/tonycox/garage/driver/ingestion/TripInfo.scala
package org.tonycox.garage.driver.ingestion

import java.time.LocalTime

/**
	* @author Anton Solovev
	* @since 17.03.2018
	*/
class TripInfo(
	val driverId: Long,
	val path: List[String],
	val departureTime: LocalTime,
	val arriveTime: LocalTime) {


}
